baseapp/baseapp.go

```diff
@@ -133,6 +133,9 @@ type BaseApp struct { // nolint: maligned
 	// indexEvents defines the set of events in the form {eventType}.{attributeKey},
 	// which informs Tendermint what to index. If empty, all events will be indexed.
 	indexEvents map[string]struct{}
+
+	// msg fee handler
+	additionalMsgFeeHandler sdk.AdditionalMsgFeeHandler // ante handler for fee and auth
```

Comment:
Since this is the core SDK and the msg fee module isn't part of the core cosmos sdk ... it feels like there shouldn't be anything named "msg fee handler" here...